SharpPcap/ICaptureHeader.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace SharpPcap
+{
+    public interface ICaptureHeader
+    {
+        uint Seconds { get; }
+        uint MicroSeconds { get; }
```

Comment:
I think a `PosixTimeval` make more sense here
```suggestion
    PosixTimeval Timeval { get; }
```